templates/repo/issue/view_content.tmpl

```diff
@@ -108,18 +108,53 @@
 							{{template "repo/issue/comment_tab" .}}
 							{{.CsrfTokenHtml}}
 							<input id="status" name="status" type="hidden">
+							<input id="state" name="state" type="hidden" >
 							<div class="field footer">
 								<div class="text right">
 									{{if and (or .HasIssuesOrPullsWritePermission .IsIssuePoster) (not .DisableStatusChange)}}
+									<!-- WIP: Use a combination of button and dropdown to implement the "close as & reopen" -->
+									<div class="ui labeled button">
 										{{if .Issue.IsClosed}}
-											<button id="status-button" class="ui green basic button" tabindex="6" data-status="{{.locale.Tr "repo.issues.reopen_issue"}}" data-status-and-comment="{{.locale.Tr "repo.issues.reopen_comment_issue"}}" data-status-val="reopen">
+											<button id="status-button" type="submit" class="ui basic button" data-status="{{.locale.Tr "repo.issues.reopen_issue"}}" data-status-and-comment="{{.locale.Tr "repo.issues.reopen_comment_issue"}}" data-status-val="reopen">
 												{{.locale.Tr "repo.issues.reopen_issue"}}
 											</button>
 										{{else}}
-											<button id="status-button" class="ui red basic button" tabindex="6" data-status="{{.locale.Tr "repo.issues.close_issue"}}" data-status-and-comment="{{.locale.Tr "repo.issues.close_comment_issue"}}" data-status-val="close">
+											<button id="status-button" type="submit" class="ui basic button" data-status="{{.locale.Tr "repo.issues.close_issue"}}" data-status-and-comment="{{.locale.Tr "repo.issues.close_comment_issue"}}" data-status-val="close">
 												{{.locale.Tr "repo.issues.close_issue"}}
 											</button>
 										{{end}}
+
+										<div id="state-dropdown" class="ui dropdown jump item tooltip gt-px-3" tabindex="-1" data-content="">
```

Comment:
I guess you need this https://fomantic-ui.com/modules/dropdown.html#floating , instead of re-inventing wheels

Reply:
yes, that's exactly what i need😂